Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -596,8 +596,8 @@ GlyphBufferAdvance Font::applyTransforms(GlyphBuffer& glyphBuffer, unsigned begi
     };
 
     auto substring = text.substring(beginningStringIndex);
-    auto upconvertedCharacters = substring.upconvertedCharacters();
-    auto localeString = locale.isNull() ? nullptr : LocaleCocoa::canonicalLanguageIdentifierFromString(locale).string().createCFString();
+    auto upconvertedCharacters = substring.upconvertedCharacters<256>();
```

Comment:
nit: maybe we could have 256 as a named constant

Reply:
Done.